data/35e/wizards_of_the_coast/supplement/dragon_magic/dm_classes.lst

```diff
@@ -11,8 +11,13 @@ CLASS:Dragonfire Adept	STARTSKILLPTS:4	CSKILL:Appraise|Bluff|Climb|Concentration
 1	ABILITY:Dragonfire Adept Class Feature|AUTOMATIC|Dragonfire Adept ~ Invocations
 1	ABILITY:Dragonfire Adept Class Feature|AUTOMATIC|Dragonfire Adept ~ Breath Weapon
 1	ABILITY:Dragonfire Adept Class Feature|AUTOMATIC|Dragonfire Adept ~ Dragontouched
-2	ABILITY:Dragonfire Adept Class Feature|AUTOMATIC|Dragonfire Adept ~ Breath Effect
+2	ABILITY:Dragonfire Adept Class Feature|AUTOMATIC|Dragonfire Adept ~ Breath Effect	BONUS:ABILITYPOOL|Dragonfire Adept Breath Effect|1
 2	ABILITY:Dragonfire Adept Class Feature|AUTOMATIC|Dragonfire Adept ~ Scales
 4	ABILITY:Dragonfire Adept Class Feature|AUTOMATIC|Dragonfire Adept ~ Dragonkin
+5		BONUS:ABILITYPOOL|Dragonfire Adept Breath Effect|1
```

Comment:
The BONUS:ABILITYPOOL should be in the ability, and formula modified:

BONUS:ABILITYPOOL|Dragonfire Adept Breath Effect|1+(DragonfireAdeptLVL/5)+(DragonfireAdeptLVL>=12)